pkg/apis/serving/v1beta1/model.go

```diff
@@ -47,6 +47,7 @@ type ModelSpec struct {
 	// +optional
 	Runtime *string `json:"runtime,omitempty"`
 
+	// TODO (TK): refactor, currently this takes the PredictorExtensionSpec for both predictor and explainer
 	PredictorExtensionSpec `json:",inline"`
```

Comment:
We should still find a solution for this (not make it Predictor or Explainer specific)